packages/regl-worldview/src/commands/GLTFScene.js

```diff
@@ -142,7 +141,7 @@ const drawModel = (regl) => {
       for (const primitive of mesh.primitives) {
         const material = model.json.materials[primitive.material];
         const texInfo = material.pbrMetallicRoughness.baseColorTexture;
-        if (!accessors || !textures) {
+        if (!accessors) {
           throw new Error("Error decoding GLB file");
```

Comment:
I think we could just remove this error entirely. The error message will be more obvious/debuggable if it says `cannot access property [x] of undefined` rather than just "Error decoding GLB". unless you want to reword the error message

Reply:
I'll reword it, I wrote it this way is so that we can maintain flow typing